components/infobox/wikis/overwatch/infobox_person_player_custom.lua

```diff
@@ -80,6 +80,10 @@ function CustomInjector:parse(id, widgets)
 			return CharacterIcon.Icon{character = CharacterNames[hero:lower()], size = SIZE_HERO}
 		end)
 
+		if #heroIcons > MAX_NUMBER_OF_SIGNATURE_HEROES then
```

Comment:
why not just
`heroIcons = Array.sub(heroIcons, 1, MAX_NUMBER_OF_SIGNATURE_HEROES)`
(without the if)